src/middleware/auth.middleware.js

```diff
@@ -0,0 +1,28 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  let userData;
+
+  try {
+    userData = jwtService.validateAccessToken(token);
+  } catch (err) {
+    res.send(err);
```

Comment:
Instead of sending the error directly with `res.send(err)`, it would be more appropriate to send a status code indicating an error, such as `res.sendStatus(401)`, to maintain consistency with other error handling in this middleware.